python/generate/default_templates/configuration.mustache

```diff
@@ -0,0 +1,626 @@
+{{>partial_header}}
```

Comment:
Was this template the correct Python flavour and version? It seems to have led to unexpected diffs in the SDK merge request.

Reply:
Quite possibly not, I took the latest one from their master branch. Let me reconsider

Reply:
Great thanks, you can use the tags to find the correct version.